### DISCUSSION TITLE
Is there a way to check length of a string with a filter?

### DISCUSSION THREAD

**Discussion by meekde** [pocketbase/pocketbase#2402] [Q&A]
`:length` seems to be reserved for arrays but is there a way to create a rule that validates string length? 

I have tried `request.data.word:length <= 80` and `request.data.word <=  80`

**Comment by ganigeorgiev** [comment]
No, at the moment there is no such filter modifier (it is in my todo and eventually will be added together with `:lower` and `:upper` modifiers).

If you want to apply a global `text` field length restriction (aka. it will apply also for admins) it might be better to set the max length as field option:

![screenshot](https://i.imgur.com/0cRUoyJ.png)
